amqp/amqp.go

```diff
@@ -0,0 +1,13 @@
+package amqp
+
+import (
+  "fmt"
+  psh "github.com/platformsh/config-reader-go/v2"
+)
+
+// AMQP requires a specfic connection string to connect to PostgreSQL.
+func FormattedCredentials(creds psh.Credential) (string, error) {
+  formatted := fmt.Sprintf("%s://%s:%s@%s:%d/", creds.Scheme, creds.Username,
+    creds.Password, creds.Host, creds.Port)
```

Comment:
Odd wrapping location.  Is that where go fmt broke it?  If not, just make it one long line.